assets/src/edit-story/components/canvas/singleSelectionMoveable.js

```diff
@@ -338,19 +338,20 @@ function SingleSelectionMoveable({
             selectedElement,
             direction,
             editorToDataX(newWidth),
-            editorToDataY(newHeight)
+            editorToDataY(newHeight),
+            delta
           );
         }
         if (updates && updates.height) {
           newHeight = dataToEditorY(updates.height);
         }
 
-        target.style.width = `${newWidth}px`;
-        target.style.height = `${newHeight}px`;
```

Comment:
And you're sure we can remove these? Also for non-text elements?

Reply:
Removed them because a few lines below we do the same thing:
```js
frame.resize = [newWidth, newHeight];
setTransformStyle(target);
```

```js
  const setTransformStyle = (target) => {
    target.style.transform = `translate(${frame.translate[0]}px, ${frame.translate[1]}px) rotate(${frame.rotate}deg)`;
    if (frame.resize[0]) {
      target.style.width = `${frame.resize[0]}px`;
    }
    if (frame.resize[1]) {
      target.style.height = `${frame.resize[1]}px`;
    }
    pushTransform(selectedElement.id, frame);
  };
```